dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgCompanyDataTable.vue

```diff
@@ -41,28 +41,28 @@
         :key="reportingPeriod.dataId"
       >
         <template #body="{ data }">
```

Comment:
Why does the #body of this `<Column>` have the value `{ data }`, and the one above (which renders the kpi field names and tooltips etc.) has the value `slotProps` ?
Can't they both be the same?

Reply:
Changed to consistent usage of `slotProps` as that enables the typescript server to recognize the allowed fields correctly (we could also use `data` directly to make it shorter but then everything is marked by the linter)

Reply:
Nice, thx.